src/pal/tests/palsuite/paltestlist.txt

```diff
@@ -532,6 +532,7 @@ file_io/DeleteFileA/test1/paltest_deletefilea_test1
 file_io/DeleteFileW/test1/paltest_deletefilew_test1
 file_io/errorpathnotfound/test2/paltest_errorpathnotfound_test2
 file_io/errorpathnotfound/test3/paltest_errorpathnotfound_test3
+file_io/filecanonicalizepath/paltest_filecanonicalizepath_test1
```

Comment:
Not sure if you weren't done addressing PR feedback, but this one is still causing the pal test failures.
```
/mnt/j/workspace/dotnet_coreclr_linux_debug_prtest/bin/obj/Linux.x64.Debug/src/pal/tests/palsuite/file_io/filecanonicalizepath/paltest_filecanonicalizepath_test1: No such file or directory
FAILED: file_io/filecanonicalizepath/paltest_filecanonicalizepath_test1. Exit code: 127
```